sources/packages/api/src-sql/AuthCodeTables/Create-initial-institution-user-types.sql

```diff
@@ -0,0 +1,23 @@
+-- Inserting initial values for 'institution_user_type_roles' table. This will be part of table creation migration. So, no drop migration sql, this data will destroy with table in drop migration
```

Comment:
Great work creating the descriptions, but it also shows that we need to duplicate a lot of the information if we are going to expand the table, as shown below. I would recommend revisiting the idea to have more two tables for users and roles types.

![image](https://user-images.githubusercontent.com/61259237/118688913-56ebe680-b7bb-11eb-9f42-91c1be6e48f1.png)

Reply:
I agree ....

Reply:
Team decision to go ahead and keep one table removing the descriptions.